index.html

```diff
@@ -139,49 +97,114 @@ <h1 class="header-title section-title">
+          <div class="setup-artifacts-cell"></div>
+
+          <div class="setup-artifacts-cell"></div>
+          <div class="setup-artifacts-cell"></div>
+          <div class="setup-artifacts-cell"></div>
+          <div class="setup-artifacts-cell"></div>
+        </div>
       </div>
-    </div>
 
-    <div class="setup-footer">
-      <div class="setup-similar hidden">
+      <div class="setup-footer">
+        <div class="setup-similar hidden">
+
+          <h4 class="setup-similar-title">Похожие персонажи</h4>
 
-        <h4 class="setup-similar-title">Похожие персонажи</h4>
+          <div class="setup-similar-list">
+            <!--Список похожих персонажей-->
+          </div>
+        </div>
 
-        <div class="setup-similar-list"><!--Список похожих персонажей--></div>
+        <button class="button setup-submit">Сохранить</button>
       </div>
+    </form>
+  </div>
 
-      <button class="button setup-submit">Сохранить</button>
-    </div>
-  </form>
-</div>
-
-<footer>
-  <p><a href="http://htmlacademy.ru" class="footer-link"><img src="img/htmlacademy-logo.png" alt="HTML Academy"
-                                                              width="130">Сделано в HTML Academy</a>в 2017 году</p>
-</footer>
-
-<template id="similar-wizard-template" style="display: none">
-  <div class="setup-similar-item">
-    <div class="setup-similar-content">
-      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 62 86" class="setup-similar-wizard">
-        <g class="wizard">
-          <use xlink:href="#wizard-coat" class="wizard-coat"></use>
-          <use xlink:href="#wizard-head" class="wizard-head"></use>
-          <use xlink:href="#wizard-eyes" class="wizard-eyes"></use>
-          <use xlink:href="#wizard-hands" class="wizard-hands"></use>
-        </g>
-      </svg>
+  <footer>
+    <p><a href="http://htmlacademy.ru" class="footer-link"><img src="img/htmlacademy-logo.png" alt="HTML Academy"
+          width="130">Сделано в HTML Academy</a>в 2017 году</p>
+  </footer>
+
+  <template id="similar-wizard-template" style="display: none">
+    <div class="setup-similar-item">
+      <div class="setup-similar-content">
+        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 62 86" class="setup-similar-wizard">
+          <g class="wizard">
+            <use xlink:href="#wizard-coat" class="wizard-coat"></use>
+            <use xlink:href="#wizard-head" class="wizard-head"></use>
+            <use xlink:href="#wizard-eyes" class="wizard-eyes"></use>
+            <use xlink:href="#wizard-hands" class="wizard-hands"></use>
+          </g>
+        </svg>
+      </div>
+      <p class="setup-similar-label">Петр Петров</p>
     </div>
-    <p class="setup-similar-label">Петр Петров</p>
-  </div>
-</template>
+  </template>
+
+  <script>
+    var fireballSize = 22;
 
-<script src="js/game.js"></script>
+    function getFireballSpeed(isWindFromLeft) {
+      if (isWindFromLeft) {
+        return 5;
+      }
+      return 2;
+    }
```

Comment:
```suggestion
return isWindFromLeft ? 5 : 2
```
критерий https://up.htmlacademy.ru/javascript/20/criteries#d17